osu.Game/OsuGame.cs

```diff
@@ -115,6 +115,34 @@ private void load(FrameworkConfigManager frameworkConfig)
 
         private ScheduledDelegate scoreLoad;
 
+        // TODO: Implement this properly as soon as the Editor is done
+        internal void LoadEditorTimestamp()
+        {
+            notifications.Post(new SimpleNotification
+            {
+                Text = @"Sorry, but this is not fully implemented yet!",
+                Icon = FontAwesome.fa_life_saver,
+            });
+        }
+
+        internal void LoadSpectatorScreen()
+        {
+            notifications.Post(new SimpleNotification
+            {
+                Text = @"Sorry, but spectating is not implemented yet!",
+                Icon = FontAwesome.fa_life_saver,
+            });
+        }
+
+        internal void LoadMultiplayerLobby(int lobbyId)

```

Comment:
Should be `LoadMatch(matchId)` if anything (the lobby is a place you find many matches)